src/avt_vimba_camera.cpp

```diff
@@ -342,9 +346,6 @@ CameraPtr AvtVimbaCamera::openCamera(std::string id_str) {
     }
   }
 
-  // set previous handler back
```

Comment:
why this is removed?
The signal() function defines the handling of the next received signal only, after which the default handling is reinstated. So it is necessary for the signal handler to call signal() if the program needs to continue handling signals using a non-default handler.

Reply:
Nice catch, this change is unintentional, I pulled in commits from PR #21 which must have changed this since the PR was based off of an old version of master.
I'll fix.